capstone/src/main/webapp/youtubeScript.js

```diff
@@ -36,42 +37,41 @@ const YOUTUBE_CATEGORIES = {
 
 /** Calls youtube servlet and passes output to perspctive */
 async function callYoutube() {
-  document.getElementById('search-type').innerHTML = ""
+  document.getElementById('search-type').innerHTML = "";
   const channelId = document.getElementById('channelIdForAnalysis').value.replace(/ /g, '');
   if (!channelId) {
     return;
   }
   /** Checks if input is a category, if so directs input to be handled by get trending*/
```

Comment:
For comments, let's use `//` instead of JSDoc

Reply:
addressed in "address Chris's and Daniels nits"